test/components/dragBoxLayerTests.ts

```diff
@@ -2,284 +2,241 @@
-      assert.deepEqual(receivedBounds.bottomRight, endPoint, "bottom-right point was set correctly");
+        assert.isTrue(callbackCalled, "the callback was called");
+        assert.deepEqual(receivedBounds.topLeft, quarterPoint, "top-left point was set correctly");
+        assert.deepEqual(receivedBounds.bottomRight, halfPoint, "bottom-right point was set correctly");
 
-      callbackCalled = false;
-      dbl.offDrag(callback);
+        callbackCalled = false;
+        dbl.offDrag(callback);
 
-      TestMethods.triggerFakeDragSequence(target, startPoint, endPoint);
-      assert.isFalse(callbackCalled, "the callback was detached from the dragoBoxLayer and not called");
+        TestMethods.triggerFakeDragSequence(target, quarterPoint, halfPoint);
+        assert.isFalse(callbackCalled, "the callback was detached from the dragoBoxLayer and not called");
+      });
 
-      svg.remove();
-    });
+      it("onDragEnd()", () => {
+        dbl.renderTo(svg);
 
-    it("onDragEnd()", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.DragBoxLayer();
-      dbl.renderTo(svg);
+        let receivedBounds: Plottable.Bounds;
+        let callbackCalled = false;
+        let callback = (b: Plottable.Bounds) => {
+          receivedBounds = b;
+          callbackCalled = true;
+        };
+        dbl.onDragEnd(callback);
 
-      let startPoint = {
-        x: SVG_WIDTH / 4,
-        y: SVG_HEIGHT / 4
-      };
-      let endPoint = {
-        x: SVG_WIDTH / 2,
-        y: SVG_HEIGHT / 2
-      };
+        let target = dbl.background();
+        TestMethods.triggerFakeDragSequence(target, quarterPoint, halfPoint);
 
-      let receivedBounds: Plottable.Bounds;
-      let callbackCalled = false;
-      let callback = (b: Plottable.Bounds) => {
-        receivedBounds = b;
-        callbackCalled = true;
-      };
-      dbl.onDragEnd(callback);
+        assert.isTrue(callbackCalled, "the callback was called");
+        assert.deepEqual(receivedBounds.topLeft, quarterPoint, "top-left point was set correctly");
+        assert.deepEqual(receivedBounds.bottomRight, halfPoint, "bottom-right point was set correctly");
+        dbl.offDragEnd(callback);
+        callbackCalled = false;
 
-      let target = dbl.background();
-      TestMethods.triggerFakeDragSequence(target, startPoint, endPoint);
+        TestMethods.triggerFakeDragSequence(target, quarterPoint, halfPoint);
+        assert.isFalse(callbackCalled, "the callback was detached from the dragoBoxLayer and not called");
+      });
 
-      assert.isTrue(callbackCalled, "the callback was called");
-      assert.deepEqual(receivedBounds.topLeft, startPoint, "top-left point was set correctly");
-      assert.deepEqual(receivedBounds.bottomRight, endPoint, "bottom-right point was set correctly");
-      dbl.offDragEnd(callback);
-      callbackCalled = false;
+      it("multiple drag interaction callbacks", () => {
```

Comment:
I thought by multiple callbacks it means that you can register more than one callback onDrag/onDragStart/onDragEnd
I think you are testing 2 things in one test case here:
one is you can register more than one callback for each event, and another is that all 3 events work together?